pkg/expression/builtin_arithmetic_vec.go

```diff
@@ -481,7 +469,7 @@ func (b *builtinArithmeticModDecimalSig) vecEvalDecimal(ctx EvalContext, input *
 	y := buf.Decimals()
 	var to types.MyDecimal
 	for i := 0; i < n; i++ {
-		if result.IsNull(i) {
+		if result.IsNull(i) || buf.IsNull(i) {
```

Comment:
Any bug here before this PR?

Reply:
No, but the previous code is strange. I make it correct.